src/System/Net/Benchmarks/Common/Shared/SingleStreamConnection.cs

```diff
@@ -3,9 +3,18 @@
 
 namespace System.Net.Benchmarks;
 
-internal abstract class SingleStreamConnection<TStream> : IConnection
+internal class SingleStreamConnection<TStream> : IConnection
```

Comment:
Nit: Can we pls revert the changes in this file as I asked in https://github.com/aspnet/Benchmarks/pull/1970/files#r1611511135?